### DISCUSSION TITLE
Supported devices Xiaomi Mini Vacuum 3C

### DISCUSSION THREAD

**Discussion by Keks1975** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#414] [General]
Good afternoon! great idea Xiaomi Cloud Map Extractor. I have Xiaomi Mini Vacuum 3C (join.vacuum.v18) firmware (4.3.3_0027). Home Assistant 2023.3.4,
vacuum cleaner parameters https://home.miot-spec.com/spec/ijey .vacuum.v18
Question, will there be support or is there an implementation for the model? Thanks.

**Comment by PiotrMachowski** [comment]
You can create a [feature request](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/new?assignees=PiotrMachowski&labels=new+platform%2Cenhancement&template=new_platform_request.yml) and hope for the best

**Reply by abdala666** [reply]
ask for the same. for Xiaomi Vacuum Mop 3C ijai.vacuum.v18. resquest send
